src/EditorFeatures/Test/InheritanceMargin/InheritanceMarginTests.cs

```diff
@@ -88,13 +88,13 @@ private static Task VerifyNoItemForDocumentAsync(string markup, string languageN
         private static async Task VerifyInheritanceMemberAsync(TestWorkspace testWorkspace, TestInheritanceMemberItem expectedItem, InheritanceMarginItem actualItem)
         {
             Assert.Equal(expectedItem.LineNumber, actualItem.LineNumber);
-            Assert.Equal(expectedItem.MemberName, actualItem.DisplayTexts.JoinText());
+            Assert.Equal(expectedItem.MemberName, actualItem.DisplayTaggedTexts.JoinText());
             Assert.Equal(expectedItem.Targets.Length, actualItem.TargetItems.Length);
             var expectedTargets = expectedItem.Targets
                 .Select(info => TestInheritanceTargetItem.Create(info, testWorkspace))
                 .OrderBy(target => target.TargetSymbolName)
                 .ToImmutableArray();
-            var sortedActualTargets = actualItem.TargetItems.OrderBy(target => target.DefinitionItem.DisplayParts.JoinText())
```

Comment:
All the change in this test file is caused by this.
It used to get a display string here, and used in the many asserts below. But given right each target/member has a display taggedText so use it for the assertion.
It only affects
1. Tooltip.
2. A corner case, when multiple members are on the same line, how to show the member's name.
After:
![image](https://user-images.githubusercontent.com/24360909/131456298-d853817c-1e15-48e6-ad48-9e9a82da4579.png)
Before:
![image](https://user-images.githubusercontent.com/24360909/131456265-68376c35-50e5-47f7-815f-2b1fc2d36548.png)
No effect on the target glyphs